src/components/NotFoundPage.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
```

Comment:
I think the name of the component should be something like `NotFound`, instead of `NotFoundPage`, since it's part of a page and not the page itself.